### DISCUSSION TITLE
Inline filters for dashboards

### DISCUSSION THREAD

**Discussion by Raghav-Arora-01** [apache/superset#26238] [Q&A / Help]
I wonder if an inline filter with 3 categories say (Month, Quarter, Year) can be there in apache superset. The working of it can be like this, if the month is selected all the visuals will show the data every month for ex jan, feb...dec.
If the quarter is selected then all the visuals show the data every quarter say quarter1 (jan-mar), quarter2 (apr-june), quarter3 (july-sept) and quarter4 (oct-dec). If year is selected then all the visuals must show the data for the whole year. 
![image](https://github.com/apache/superset/assets/55590803/e12ac587-1a69-4fe6-860e-2c5dd06814cf)

**Comment by cyzchenyuzhu** [comment]
同问，关于日期过滤需要以上功能
